Dockerfile

```diff
@@ -330,7 +330,7 @@ ENV ANDROID_HOME=/opt/android-sdk
 COPY --from=androidbuild /usr/bin/repo /usr/bin/
 COPY --from=androidbuild /etc/profile.d/android.sh /etc/profile.d/
 COPY --chown=$USERNAME:$USERNAME --from=androidbuild ${ANDROID_HOME} ${ANDROID_HOME}
-RUN chmod o+rw ${ANDROID_HOME}
+RUN chmod -R o+rw ${ANDROID_HOME}
```

Comment:
Commit message: This doesn't only affect GitHub actions, it also affects all other users of the image, right?
Maybe just remove the `in github actions` part of the commit message.

Reply:
Ok, changed the commit message